app/src/test/java/com/github/sdp/tarjetakuna/ui/webapi/WebApiTest.kt

```diff
@@ -22,4 +22,18 @@ class WebApiTest {
         assert(api.getSets().isExecuted.not())
         assert(api.getCards().isExecuted.not())
     }
+
+    @Test
+    fun test_getCards(){
+        val fCards = WebApi.getCards()
+        fCards.whenComplete { cards, throwable ->

```

Comment:
Question: why not use thenAccept and exceptionally?

Reply:
because it was easier to test for throwable in this case in order to assert an exception in case of failure.
In actual code (not in test), thenAccept + exceptionnally are better to use